packages/react/src/box/styles.ts

```diff
@@ -531,14 +531,24 @@ export const linkStyles = {
 };
 
 type FocusProps = Partial<{
-	/** If true, the focus indicator will be shown when the element receives keyboard focus. */
+	/** @deprecated use focusRingFor="keyboard". */
 	focus: boolean;
+	/** Display a focus indicator when the element receives focus. 'all' includes programmatic focus and 'keyboard' is for keyboard only focus*/
+	focusRingFor: 'all' | 'keyboard';
 }>;
 export const focusStyles = {
```

Comment:
I was going to suggest removing these vars, but then I saw that we export and use these styles in a few places. I wonder if we can not use these raw in other components in a future PR?

Reply:
I think it's more about whether or not consumers have used it. Since we `export *`, this is bundled in and a change would be a `major` release.